### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by lannisterrr**
badge

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  width: 3rem;
+  height: 3rem;
+}
+
+.badge-avatar-red {
+  background: var(--primary-color);
+}
+
+.badge-avatar-green {
+  background: green;
```
Comment (bhanuprakashj) [inline components/badge/badge.css:24]: Use these colors from variables with its hex code ideally

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.badge-avatar-grey {
+  background: grey;
```
Comment (bhanuprakashj) [inline components/badge/badge.css:28]: Use these colors from variables with its hex code ideally

File: components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  height: 1rem;
+  border: 0.125rem solid var(--secondary-color);
+}
+
+/* badge-icon */
+
+.badge {
+  background: var(--primary-color);
+  border-radius: 50%;
+  color: white;
```
Comment (bhanuprakashj) [inline components/badge/badge.css:45]: Use these colors from variables with its hex code ideally

File: components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+@import url('../../starterFiles/abstracts/variables.css');
+@import url('../../starterFiles/base/reset.css');
+@import url('../../starterFiles//utils/util.css');
```
Comment (junaidshaikh-js) [inline components/badge/badge.css:3]: The double slash ( // ) in the import url is by mistake, I guess. Remove it. Also for line 4 and 5

File: components/badge/badge.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="badge.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
```
Comment (junaidshaikh-js) [inline components/badge/badge.html:10]: You can import fontawesome in your CSS so you don't have to import it multiple times on every HTML page. 

Also, you can use a single CSS file for all the HTML files, import all your component CSS into one file, and attach it to all the HTML that way you will get the error if by mistake overwrite somewhere.

File: components/badge/badge.html [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    />
+    <title>avatar</title>
```
Comment (junaidshaikh-js) [inline components/badge/badge.html:12]: Title can be Sentence case (Avatar)

**Review by junaidshaikh-js — COMMENTED** [review]
Move your badge to the left slightly on your avatar.

File: components/badge/badge.css (hunk 4 above)
Comment (lannisterrr) [reply components/badge/badge.css:3]: no it's the path for the files.

**Pull request merged**